js/src/views/Settings/Advanced/advanced.js

```diff
@@ -0,0 +1,105 @@
+// Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component } from 'react';
+import LogLevel from 'loglevel';
+
+import { Container, Select } from '~/ui';
+import { LOG_KEYS } from '~/config';
+
+import layout from '../layout.css';
+
+export default class AdvancedSettings extends Component {
```

Comment:
As per above, needs to go onto pre-existing tab. In addition since there is actually no use for it currently (one single modal), it is a feature that shouldn't be show in NODE_ENV === production.